setup.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine
+COPY . /home/src
+WORKDIR /home/src
+RUN pip3 install requests
+
+ENTRYPOINT ["python", "setup.py"]
```

Comment:
I think this should be 
`ENTRYPOINT ["python", "setup.py", "--docker"]`

Reply:
```suggestion
ENTRYPOINT ["python", "setup.py", "--docker”]
```